File: fib04.clj
(set! *unchecked-math* true)

(defn fib ^long [^long n]
  (if (< n 3)
    1
    (+ (fib (dec n)) (fib (- n 2)))))

(defn fibTest []
  (println (fib 44))
  (println (fib 45))
  (println (fib 46)))

(time (fibTest))

File: fib04.dart
int fib(int n) 
{
   if (n < 2) 
     return n;
   return fib(n - 1) + fib(n - 2);
}

main() 
{
  for (int i = 1; i <= 3; i++) 
  {
    var f = fib(46);
    print("$i = $f");
  }
}
